README.md

```diff
@@ -14,9 +14,9 @@ Kubeadm's scope is limited to the local node filesystem and the Kubernetes API,
 
 ## Common Kubeadm cmdlets
 1. **kubeadm init** to bootstrap the initial Kubernetes control-plane node.
-1. **kubeadm join** to bootstrap a Kubernetes worker node or an additional control plane node, and join it to the cluster.
-1. **kubeadm upgrade** to upgrade a Kubernetes cluster to a newer version.
-1. **kubeadm reset** to revert any changes made to this host by kubeadm init or kubeadm join.
+2. **kubeadm join** to bootstrap a Kubernetes worker node or an additional control plane node, and join it to the cluster.
+3. **kubeadm upgrade** to upgrade a Kubernetes cluster to a newer version.
+4. **kubeadm reset** to revert any changes made to this host by kubeadm init or kubeadm join.
```

Comment:
This is not necessary, markdown will do it automatically, and hard coding here will increase our maintenance cost.
See https://github.com/kubernetes/kubeadm#common-kubeadm-cmdlets

Reply:
Thanks, I got it